eas.json

```diff
@@ -46,6 +46,28 @@
         "key": "eas-1"
       }
     },
+    "qa": {
+      "channel": "qa",
+      "distribution": "internal",
```

Comment:
can you change distribution for 'staging' to 'store' now?

Reply:
Yes, in case we're going to publish these via the App Store, otherwise I would set this as internal.
I'll change it to store, and we can always revisit this for any project needs.

Reply:
Changed to "store" per our conversation about this.
Staging builds might be distributed via the store, therefore "store" is the right way